games/2048.cpp/Portfile

```diff
@@ -0,0 +1,35 @@
+PortSystem              1.0
+
+name                    2048.cpp
+# no official releases
+version                 0
+# revision number is the commit count
+revision                354
+categories              games
+maintainers             {mails.ucas.ac.cn:chenguokai17 @chenguokai} openmaintainer
+
+homepage                https://github.com/plibither8/2048.cpp 
+platforms               darwin
+license                 MIT
+fetch.type              git
+git.url                 https://github.com/plibither8/${name}.git
+git.branch              81c0eebaa2e575c7c7943c01958b88896c9da852 
```

Comment:
You should use the `github 1.0` portgroup instead.

Reply:
All fixed, thanks for your review!